frontend/src/components/ViewActivities/editActivityFormElements.js

```diff
@@ -10,13 +10,13 @@ const TIMEWIDTH = 2;
 const TZPICKERWIDTH = 3;
 
 /**
- * Create a Text element Form Group for the editActivity form.
- *
- * @param {string} controlId FormGroup's control ID.
- * @param {string} formLabel The label of the field for this FormGroup.
- * @param {string} placeHolder The input's placeholder.
- * @param {ref} ref The input's reference.
- * @returns {HTML} A text element form group.
+ * Create a Text element Form Group for the editActivity form. 
+ * 
+ * @param {!string} controlId FormGroup's control ID.
+ * @param {!string} formLabel The label of the field for this FormGroup. 
+ * @param {!string} placeHolder The input's placeholder. 
+ * @param {?ref} ref The input's reference.
+ * @return {HTML} A text element form group.
```

Comment:
I am not sure if this is more accurate or not, but when I return React stuff like this (not vanilla HTML) I use JSX.Element. You can keep this but thought I would mention if you want to use it too.
```suggestion
 * @return {JSX.Element} A text element form group.
```